sease.markdown

```diff
@@ -0,0 +1,7 @@
+
+---
+name: Sease
+# upload your logo to the following directory - must be square
+logo: '[/assets/media/partners/sease.png](https://sease.io/wp-content/uploads/2022/09/SEASE-FAVICON-gradient-24-4.png)'
```

Comment:
Please upload your logo as part of this pull request, not hyperlinked over to an external location.

Reply:
Hello, we have done this but it is still giving error. Could you please help me fix it? Thank you!